nixos/modules/hardware/video/nvidia.nix

```
@@ -460,10 +472,29 @@ in
           #   "nvidia" driver.
           # - Add the AllowEmptyInitialConfiguration option to the Screen section for the
           #   "nvidia" driver, in order to allow the X server to start without any outputs.
-          # - Add a separate Device section for the Intel GPU, using the "modesetting"
-          #   driver and with the configured BusID.
-          # - OR add a separate Device section for the AMD APU, using the "amdgpu"
-          #   driver and with the configures BusID.
+          # - Ad the `BusID`s to the configuration of the iGPU `Device` sections.
```

```suggestion
          # - Add the `BusID`s to the configuration of the iGPU `Device` sections.
```

Isn't there a `d` missing?

---

Defininitely, thanks!